### ISSUE TITLE
Skip points: Exclude bar-x.. bar-y completely from project (including render)

### ISSUE THREAD

**Issue by musikBear** [LMMS/lmms#1175]
This idea came to me as i was looking at an older prroject. I realized that the piece would be better if a number of bars was removed from the middle of the project.
The idea was -What if i could 'fold' sondeditor so (lets say) bar-16 would continue, not in bar-17, but in bar-24. That may sound as a difficult thing, but it actually only a rule for the play-head.
Visually is a different thing. Ofcause it would look sexy is the SE actually did fold :p, but a simple 'gray-out' of the whole unwanted area, would do fine.
Rule-wise could be implemented with block-mark(s) on timeline.
The benefit is ofcause that it is possibel to make a major change to the project, very fast, and very easy regret that change. It would allow the user to do easy tests, with different sub-sections (without deleting and moving a lot of blocks in  project sub-version(s).

**Comment by tresf** [comment]
Like an export black-out feature?  It would be more of a "skip ahead" instead of a loop point in that case.  How about you change the request to add an additional loop point mode where the loop points turn into skip points and the cursor jumps/teleports to a different part of the song.  Then, you could export with loop points enabled as a black-out zone.

-Tres

**Comment by Sti2nd** [comment]
> Then, you could export with loop points enabled as a black-out zone.

From the beginning, but not if you want it to export from anywhere else and skip something. That feature was removed.

**Comment by tresf** [comment]
> From the beginning, but not if you want it to export from anywhere else and skip something. That feature was removed.


No, that bug was fixed.  You can use loop points to do as you are describing please don't be dramatic.

**Comment by Sti2nd** [comment]
So you are suggesting four loop points? Ok :-)

**Comment by tresf** [comment]
> So you are suggesting four loop points? Ok :-)

No.

**Comment by Spekular** [comment]
If we allow labeling regions in the song editor we could allow to render/exclude selected regions only.

**Comment by tresf** [comment]
> If we allow labeling regions in the song editor we could allow to render/exclude selected regions only.

Good point.  Possible overlap in functionality with #156.

**Comment by Spekular** [comment]
Perhaps this and #709 could be merged somehow? They're not quite duplicates, but very close.
